inference-engine/include/vpu/vpu_config.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Should this file be hidden from user?

Reply:
We have discussed it with @taka-no-me. 
The result is such that no public API can be in `details` or hidden from the user

Reply:
But is `vpu_config.hpp` is a part of public API? I though it is just a place for common options which is not intended to be used.

Reply:
@gladilov-gleb it is more an API itself than its implementation detail. The identifiers of config options declared in this header are supposed to be used in user code therefor header is not supposed to be "hidden" in details folder.